tools/qm-storage-settings

```diff
@@ -38,18 +37,18 @@ exec_command() {
     fi
 
     # print the output of QM ROOTFS
-    echo $($COMMAND)
```

Comment:
I believe the idea here was just print, not execute. With the result of this echo the other function would do something.

Reply:
Look here 
some how it returned as an input value and the output
19:28:08             out: Exiting... unable to find /usr/share/qm/qm-rootfs/etc/containers/storage.conf

Reply:
I would just leave the bash header as it's and this one as well but I will check tomorrow. I might be little "snow" as it's holiday here but I will check.